src/mesh/parallel/identity.cxx

```diff
@@ -0,0 +1,11 @@
+#include "bout/paralleltransform.hxx"
```

Comment:
I think we should really treat `identity` as any other parallel transform rather than including the class definition in the general `paralleltransform.hxx`.  See 4135a74 of #1464 and related commits for an example of what this looks like. I just mention as adding this file is a part of those changes so it might be worth doing that split now.